arduino101load/arduino101copy.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+    "io"
+    "log"
+    "os"
+    "fmt"
+)
+
+func main() {
+    args := os.Args[1:]
+
+    // Make sure there are only two arguments
+    if len(args) != 2 {
+        log.Fatal("copy.exe <source> <destination>")
+    }
+    
+    source_path := args[0]
+    destination_path := args[1]
+
+    // Make sure the source file exists
+    source_file, err := os.Open(source_path)
+    exit_if_error(err)
+    defer source_file.Close()
+    
+    // Make sure we can create the destination file
+    destination_file, err := os.Create(destination_path)
```

Comment:
Looks like you're creating an empty version of the dest. file in order to check if you can access that path? is there a cleaner way to do that?